app/views/logged_in.erb

```diff
@@ -0,0 +1,10 @@
+<span id='logged_in'>
+  <form action='/sessions/<%= session[:user] %>' method='post'>
+    <b>You're Logged In As: <%= session[:name] %>!</b>
+    <br>
```

Comment:
1. HTML is for structure *only*. Sometimes - very rarely, but sometimes - `<br />` is used structurally e.g. poetry -> presentation stuff should happen in CSS. If you want a gap, set `display: block;` on the element you want to linebreak.

2. If using a self-closing tag (HTML4...) **explicitly** close the tag
```erb
<br> -> <br />
<img />
<input />
```